app/mailers/visitor_mailer.rb

```diff
@@ -21,27 +21,48 @@ def booking_confirmation_email(visit, confirmation, token)
     @confirmation = confirmation
     @token = token
 
-    mail(from: sender, reply_to: prison_mailbox_email, to: recipient, subject: "Visit confirmed: your visit for #{Date.parse(@slot.date).strftime('%e %B %Y').gsub(/^ /,'')} has been confirmed")
+    mail(from: sender,
+         reply_to: prison_mailbox_email,
+         to: recipient,
+         subject: "Visit confirmed: your visit for \
```

Comment:
I was wondering about that, and meant to ask. There are a whole bunch of these; we should check before merging.